zomes/likes/code/src/hdk_membrane.rs

```diff
@@ -0,0 +1,10 @@
+struct MembraneObject {
```

Comment:
calling stuff `hdk_` here in app-spec is confusing to me.
Is this suppose to be in `hdk-rust` ?

Reply:
Yes.

Reply:
Okay, so more points for merging app-spec into hdk-rust